jest.config.ts

```diff
@@ -1,5 +1,7 @@
 import { getJestProjects } from '@nx/jest'
 
-export default {
+const jestConfig = {
   projects: getJestProjects(),
 }
+
+export default jestConfig
```

Comment:
o.O

Not opposed to the change, but this seems to amount to the same thing in the end. No?

Reply:
yes, but is honoring this rule https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-anonymous-default-export.md